coalib/bears/Bear.py

```diff
@@ -47,6 +47,13 @@ class Bear(Printer, LogPrinter):
     >>> class SomeBear(Bear):
     ...     LANGUAGES = "Java"
 
+    To see what requirements the bear has, give the bear a value for
+    ``REQUIREMENTS``, which will be a tuple of strings:
+
+    >>> class SomeBear(Bear):
+    ...     REQUIREMENTS = ('yamllint~=1.0.1')
+    ...     REQUIREMENTS == SomeBear.requirements
```

Comment:
though I would remove the prop below, you can actually test this with writing below `True`
-->
```python
>>> SomeBear.REQUIREMENTS == SomeBear.requirements
True
```
but then this needs to be outside of the class :+1: